### ISSUE TITLE
JumpLinks - aria attributes need correction

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly#7286]
**Describe the enhancement or change**
Various aria attributes that need some form of correction in examples/demos:

- `aria-current="location"` needs to be applied to the `li` that is currently selected in a nav
- The `nav` and `ul` elements that makes up the navigation in the component should have unique aria labeling - either `aria-label` if there's no visible text label (i.e. the "Jump to section" text) or `aria-labelledby` to point to that visible text. This would require providing unique label text for the 'main' jump links, or providing unique nav text if there are nested lists (like in the "Vertical with inactive subsections" example, since the nested list would make sense to be labeled by its parent nav item).
- For the expandable implementation, the toggle button shouldn't have an aria-label if there's visible text to use. This will be confusing since the aria-label text isnt visible anywhere and doesn't match the text actually used in the button. In react it is possible to have an expandable JumpLinks without a text label in which an aria-label would make sense, but this depends if we intended that to be supported.

**Any other information?**
